### DISCUSSION TITLE
remove max-width from settings dialoge

### DISCUSSION THREAD

**Discussion by ferengi82** [home-assistant/frontend#22746] [Accessibility]
hello,

for large screens, the columns in the config page are currently limited to 1000 px by the max-width parameter, which creates a lot of empty space and the columns are unnecessarily narrow. This causes the text to be cut off for some values.

Since I have not yet thought of a reason why the max-width parameter could be useful here, I wanted to suggest removing it or at least increasing it so that the full area can also be used on 4k screens.

<img width="1726" alt="current Config Page" src="https://github.com/user-attachments/assets/e44c9c32-c70d-47c1-b417-2bed8863fca5">
<img width="1718" alt="without max-width" src="https://github.com/user-attachments/assets/85f57c51-473e-4623-81f7-983197cac895">

**Comment by ildar170975** [comment]
As a workaround, can be fixed locally: https://community.home-assistant.io/t/allow-columns-in-device-info-settings-to-be-resized-or-scale-to-screen-width/703249/6?u=ildar_gabdullin

**Comment by westi2000** [comment]
@ferengi82  ich habe ebenso diese schwarzen Ränder, wo kann man den Code entfernen "max-width"?
